summit.proto

```diff
@@ -855,4 +855,48 @@ message AdaptiveState
 	double Prog2AmpInMilliamps = 3;
 	double Prog3AmpInMilliamps = 4;
 	double RateTargetInHz = 5;
+}
+
+enum AdaptiveTherapyState 
+{
+	State0 = 0; 
+	State1 = 1;
+	State2 = 2; 
+	State3 = 3;  
+	State4 = 4; 
+	State5 = 5;  
+	State6 = 6; 
+	State7 = 7;
+	State8 = 8; 
+	StateNotSet = 15;  	
+}
+
+enum AdaptiveTherapyStatus

```

Comment:
AdaptiveTherapyStatus and AdaptiveTherapyModes are basically identical. this must have slipped through the review process on the medtronic side during development. if it's providing the same information, be probably don't have to support both, but it wouldn't be wrong to include it.